ci_framework/roles/repo_setup/tasks/install.yml

```diff
@@ -1,19 +1,27 @@
 ---
+- name: Ensure directories are present
+  ansible.builtin.file:
+    path: "{{ cifmw_repo_setup_basedir }}/{{ item }}"
+    state: directory
+  loop:
+    - tmp
+    - artifacts/repositories
+
 - name: Get repo-setup repository
   ansible.builtin.git:
     accept_hostkey: true
-    dest: "{{ cifmw_repo_setup_basedir }}/repo-setup"
-    repo: "https://github.com/ci-framework/repo-setup"
+    dest: "{{ cifmw_repo_setup_basedir }}/tmp/repo-setup"
+    repo: "https://github.com/openstack-k8s-operators/repo-setup"
 
 - name: Initialize python venv and install requirements
   ansible.builtin.pip:
     virtualenv: "{{ cifmw_repo_setup_basedir }}/venv"
-    requirements: "{{ cifmw_repo_setup_basedir }}/repo-setup/requirements.txt"
+    requirements: "{{ cifmw_repo_setup_basedir }}/tmp/repo-setup/requirements.txt"
```

Comment:
why the tmp addition?

Reply:
I'd like to keep the basedir clean. If it causes an issue, I may create a new parameter, defaulting to basedir/tmp, so that you may override it if it doesn't suit your usage?